README.md

```diff
@@ -1,53 +1,131 @@
+
+GIBS expects to recieve images in one of three Coordinate Reference System (CRS) projections.
+
+|Region     |Code      |Name                                                     |
+|---        |---       |---                                                      |
+|north polar| EPSG:3413|WGS 84 / NSIDC Sea Ice Polar Stereographic North         |
+|south polar| EPSG:3031|WGS 84 / Antarctic Polar Stereographic                   |
+|global     |EPSG:4326 |WGS 84 -- WGS84 - World Geodetic System 1984, used in GPS|
+
+HyBIG processing will attempt to choose a GIBS-suitable target CRS from the
+input image or read it from the inputs.  Reprojection is done by resampling via
+nearest neighbor. It is important to note that HyBig output are not scientific
+data, but browse imagery.
+
+
+### Tiling
+
+Large output images are divided into smaller, more manageable tiles for
+efficient handling and processing, as per agreement with GIBS. The maximum
+untiled image size generated by HyBIG is 67,108,864 cells (8,192 x 8,192). If
+the output image exceeds this threshold, HyBIG automatically tiles the output
+into multiple 4,096 x 4,096 cell images. The tiles at the edges are truncated
+to fit the overall image dimensions.  Currently you cannot override this
+behavior.
+
+### Coloring
+
+HyBIG images are colored in several ways. A palette can be included in the
+input [STAC
+Item](https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md). If
+an Item's asset contains a value with the role of `palette`, it is assumed to
+be a reference to a remote color table, which is fetched from the asset's
+`href` and parsed as a GDAL color table.
+
+If the STAC Item lacks color information, the Harmony message source is
+searched for a related URL with a "content type" of `VisualizationURL` and a
+"type" of `Color Map`. If found, it is presumed to be a remote color table and
+fetched from that location.
+
+In the absence of remote color information, the input image itself is searched
+for a color map, which is used if present.
+
+If no color information can be found, grayscale is used.
+
+### Customizations
+
+HyBIG tries to provide GIBS-appropriate defaults. You can request user
+customizations such as `crs`, `scale_extents`, `scale_sizes`, and dimensions
+(`height` & `width`) in the harmony request. However, the generated outputs may
+not be compatible with GIBS.
+
+If you specify `scale_extent` or `scale_size`, you must also include a `crs` in
+the request, and the values must match the target CRS. For example, specifying
+a bounding box in degrees requires a target CRS also with units of degrees.
+
+
+
 ## Repository structure:
 
 ```
-|- .pre-commit-config.yaml
+|- 📂 bin
+|- 📂 docker
+|- 📂 docs
+|- 📂 harmony_browse_image_generator
+|- 📂 tests
 |- CHANGELOG.md
 |- CONTRIBUTING.md
+|- LICENSE
 |- README.md
-|- bin
 |- conda_requirements.txt
 |- dev-requirements.txt
-|- docker
-|- docs
-|- harmony_browse_image_generator
+|- legacy-CHANGELOG.md
```

Comment:
Thanks for adding these missing items. (Although, now it's making me realise we're potentially missing the description of the license file in the list below, maybe the file is self-explanatory, but might be worth mentioning it's required for NASA open-source release)

Reply:
I think if we were to capture anything it would be that the license is in compliance with NASA open-source software requirements. Potential text:

> * `LICENSE` - Required for distribution under NASA open-source approval. Details conditions for use, reproduction and distribution.

Probably not perfect text, but I think that's the gist of anything I would add.